CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
-[Unreleased]: https://github.com/rust-marker/marker/releases/tag/v0.2.1...HEAD
+[Unreleased]: https://github.com/rust-marker/marker/releases/tag/v0.3.0...HEAD
```

Comment:
The link here was wrong, `/releases/tag` should be replaced with `/compare`. I fixed it in https://github.com/rust-marker/marker/pull/255.

Btw. you should leave the `[Unreleased]` section. Just move all that was previously unreleased under `0.3.0` and make the new `## [Unreleased]` section empty for now